File: templates/Content/BenchmarkDotNet.BenchmarkProjectTemplate.CSharp/BenchmarkConfig.cs
using BenchmarkDotNet.Analysers;
using BenchmarkDotNet.Attributes;
using BenchmarkDotNet.Columns;
using BenchmarkDotNet.Configs;
using BenchmarkDotNet.Exporters;
using BenchmarkDotNet.Exporters.Csv;
using BenchmarkDotNet.Jobs;
using BenchmarkDotNet.Loggers;

namespace _BenchmarkProjectName_
{
    public class BenchmarkConfig : ManualConfig
    {
        public BenchmarkConfig()
        {
            // Configure your benchmarks, see for more details: https://benchmarkdotnet.org/articles/configs/configs.html.
            //Add(Job.Dry);
            //Add(ConsoleLogger.Default);
            //Add(TargetMethodColumn.Method, StatisticColumn.Max);
            //Add(RPlotExporter.Default, CsvExporter.Default);
            //Add(EnvironmentAnalyser.Default);
        }
    }
}

File: templates/templates/BenchmarkDotNet.BenchmarkProjectTemplate.VB/BenchmarkConfig.vb
Imports BenchmarkDotNet.Analysers
Imports BenchmarkDotNet.Attributes
Imports BenchmarkDotNet.Columns
Imports BenchmarkDotNet.Configs
Imports BenchmarkDotNet.Exporters
Imports BenchmarkDotNet.Exporters.Csv
Imports BenchmarkDotNet.Jobs
Imports BenchmarkDotNet.Loggers

Namespace _BenchmarkProjectName_
    Public Class BenchmarkConfig
        Inherits ManualConfig

        Public Sub New()
            ' Configure your benchmarks, see for more details: https://benchmarkdotnet.org/articles/configs/configs.html.
            ' Add(Job.Dry);
            ' Add(ConsoleLogger.Default);
            ' Add(TargetMethodColumn.Method, StatisticColumn.Max);
            ' Add(RPlotExporter.Default, CsvExporter.Default);
            ' Add(EnvironmentAnalyser.Default);
        End Sub
    End Class
End Namespace
